src/ts/settlement.ts

```diff
@@ -91,6 +91,12 @@ export type EncodedSettlement = [
  */
 export const FULL_FEE_DISCOUNT = 10000;
 
+/**
+ * Maximum number of trades that can be included in a single call to the settle
+ * function.
+ */
+export const MAX_TRADES_IN_SETTLEMENT = 255;
```

Comment:
Do you think this is too small? We could use 2 bytes for the trade count for a whooping 4 gas/settlement more but for a bit of peace of mind. Currently, we can settle 110  orders (without any interactions), so it the block gas limit increasing by a factor of 2 and change is already enough to make this a limiting factor?

Reply:
Agreed. i don't think we'll ever use the second byte, but the cost is really small enough that it's worth it.